src/entities/base-no-id.entity.ts

```diff
@@ -0,0 +1,27 @@
+import {
+    CreateDateColumn,
+    JoinColumn,
+    ManyToOne,
+    PrimaryGeneratedColumn,
+    UpdateDateColumn,
+} from "typeorm";
+
+/**
+ * This is similar to the base entity class but without the id column. This allows classes to use
+ * (composite) keys
+ */
+export abstract class DbBaseNoIdEntity {
```

Comment:
As far as I can see, we're not using this. Remove until we need it.

Reply:
Yes, I forgot to delete it.